Zilon.Core/Zilon.Bot.Players.DevelopmentTests/BotActorTaskSourceTests.cs

```diff
@@ -25,25 +25,98 @@ public class BotActorTaskSourceTests
         [Parallelizable(ParallelScope.All)]
         public async Task GetActorTasksTestAsync(string mode)
         {
-            var _globalServiceContainer = new ServiceCollection();
+            var serviceContainer = new ServiceCollection();
             var startUp = new Startup();
-            startUp.RegisterServices(_globalServiceContainer);
-            var _globalServiceProvider = _globalServiceContainer.BuildServiceProvider();
+            startUp.RegisterServices(serviceContainer);
+            var serviceProvider = serviceContainer.BuildServiceProvider();
 
             var botSettings = new BotSettings { Mode = mode };
 
             var autoPlayEngine = new AutoplayEngine<HumanBotActorTaskSource>(startUp, botSettings);
-            await autoPlayEngine.StartAsync(_globalServiceProvider).ConfigureAwait(false);
 
-            var scoreManager = _globalServiceProvider.GetRequiredService<IScoreManager>();
+            var startPerson = autoPlayEngine.CreateStartPerson(serviceProvider);
+
+            PrintPersonBacklog(startPerson);
+
+            await autoPlayEngine.StartAsync(startPerson, serviceProvider).ConfigureAwait(false);
+
+            PrintResult(serviceProvider);
+        }
+
+        private void PrintPersonBacklog(HumanPerson humanPerson)
+        {
+            Console.WriteLine("Build In Traits:");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310821)